.github/workflows/release_next.yml

```diff
@@ -171,12 +176,26 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-node@v3
         with:
-          python-version: 3.x
-      - run: pip install -r docs/pip_requirements.txt
-      - run: mkdocs gh-deploy --force -m "[ci skip]"
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm install --frozen-lockfile
         working-directory: docs
+        
+      - name: Test build docs
+        run: npm run build
+        working-directory: docs
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
Is there no github-owned package for publishing pages?

Reply:
Didn't find anything simular directly. Docusaurus suggests to work with this popular third-party deployment action 

https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus

Reply:
How about this? https://github.com/actions/deploy-pages

Reply:
Nice find, that appears to work flawlessly 🤩

Reply:
https://github.com/Jwaegebaert/cli-build-test/actions/runs/4779007444/jobs/8495802534

Reply:
Nice find. That appears to work as well 😄 

https://github.com/Jwaegebaert/cli-build-test/actions/runs/4779137491

Although it looks like it doesn't create a commit to the branch `gh-pages`. Is that common for GitHub action deploys to pages?

Reply:
Good question. Let's find out

Reply:
Instead of deploying the build to a separate branch, they do a deployment to a GitHub Environement. https://github.com/Jwaegebaert/cli-build-test/deployments

Reply:
Just to clarify: that's sufficient for our purposes, right? I can see the docs on your repo just fine (excl. the mismatched URL but that's deliberate) but not sure if that's from a previous deployment or not.

Reply:
That should be sufficient enough. My docs is just because of the quick setup 😅